src/constructs/ec2/security-groups.ts

```diff
@@ -65,8 +83,8 @@ export class GuPublicInternetAccessSecurityGroup extends GuSecurityGroup {
   constructor(scope: GuStack, id: string, props: SecurityGroupProps) {
     super(scope, id, {
       ...props,
-      ingresses: transformToCidrIngress(Object.entries({ GLOBAL_ACCESS: "0.0.0.0/0" })),
-      description: `Allows internet access on ${GuPublicInternetAccessSecurityGroup.defaultIngressPort.toString()}`,
+      ingresses: [{ range: Peer.anyIpv4(), port: 443, description: "Allows internet access on 443" }],
+      description: "Allows internet access on 443",
```

Comment:
Minor: there are a couple of descriptions where we mention allowing access over 443, but I think that's already reasonably clear from the port definition. Perhaps mentioning allowing traffic over HTTPS would add more helpful information for anyone who doesn't know what this port is used for?

Reply:
Great suggestion. Done.